src/OrleansTestingHost/Extensions/TestConfigurationExtensions.cs

```diff
@@ -14,37 +14,40 @@ public static class TestConfigurationExtensions
         /// <summary>
         /// This call tweaks the cluster config with settings specific to a test run.
         /// </summary>
-        public static void AdjustForTestEnvironment(this ClusterConfiguration clusterConfig)
+        public static void AdjustForTestEnvironment(this ClusterConfiguration clusterConfig, string dataConnectionStringFallback)
```

Comment:
Ah, I see.  I'll leave this to your discression, but I'd hope we've a better means of overriding the settings than including each setting we want to change in this calls arguements.